src/decoder/builder.rs

```diff
@@ -55,9 +59,7 @@ pub struct Settings {
     /// Can be obtained from file metadata or by seeking to the end of the stream.
     pub(crate) byte_len: Option<u64>,
 
-    /// Whether to use coarse seeking.
-    /// Coarse seeking is faster but less accurate: it may seek to a position slightly before or
```

Comment:
> /// Coarse seeking is faster but less accurate: it may seek to a position slightly before or after the requested one, especially when the bitrate is variable.

was this not true? I like to have it documented here if it is true

Reply:
It is but it's duplicate with the documentation of `fn with_coarse_seek`. I chose to remove the duplicate parts, and instead mention here that Rodio's accurate seeking means "sample-accurate".

Reply:
oh its in `fn with_coarse_seek`, that a better place for it :+1: